src/components/ImageHeader.js

```diff
@@ -21,11 +21,14 @@ const ImageHeader = ({ image, title }: Props) => (
       }}
       style={styles.contentPadding}
     >
-      {(Array.isArray(title) ? title : [title]).map(line => (
-        <Text key={line} type="h1" style={styles.title}>
-          {line}
-        </Text>
-      ))}
+      {(Array.isArray(title) ? title : [title]).map((line, index, all) => {
+        const zIndex = all.length - index;
+        return (
+          <Text key={line} type="h1" style={[{ zIndex }, styles.title]}>
```

Comment:
I'm confused - How is zindex being used here? I don't see the corresponding styles

Reply:
style takes either a Stylesheet style, a style object (used for dynamic styles) or an array of either. zIndex is part of a style object.